### DISCUSSION TITLE
update config file

### DISCUSSION THREAD

**Discussion by francescoblefari** [vectordotdev/vector#14474] [Q&A] [answered]
Hi, I'm using helm. Can I update the config file? If yes, What's the method? Thank you.

**Comment by jszwedko** [comment, marked as answer]
Hi @francescoblefari ,

You can provide custom configuration via the `customConfig` value. See https://github.com/vectordotdev/helm-charts/tree/develop/charts/vector for more details.

**Reply by francescoblefari** [reply]
can u show me how i can do it?

**Reply by jszwedko** [reply]
> can u show me how i can do it?

Which part are you having trouble with? You can see an example in the README I linked.

**Reply by francescoblefari** [reply]
Thanks for your help.
